### ISSUE TITLE
Violation of Binance rate limit in fixed grid strategy

### ISSUE THREAD

**Issue by abduramann** [hummingbot/hummingbot#5473]
**Describe the bug**
In fixed grid strategy, Binance rate limit (#order per second) is violated when the number of levels is high.

**Steps To Reproduce**
1. Add a Binance connection.
2. Create a fixed grid bot with n_levels > 90.
3. Start the bot
4. While the bot is creating the orders, some orders cannot be created because of the rate limit.

**Release version**
1.5.0

**Solution:**
Either add a sleep time between the orders or (better) add a config parameter since this problem is not specific to Binance or Binance may alter this limit.

**Comment by abduramann** [comment]
Found a solution: Changing `rate_limits_share_pct` parameter in conf_global.yml to a value less than 50.

I think this is a connector issue. Currently, hummingbot assumes that the limit of Binance is 10 orders per second but Binance returns "50 orders per 10 seconds" error.

**Comment by abduramann** [comment]
This is getting worse day-by-day. The responsiveness of command shell is so low and now history command does not work and I get this error:

```
2022-07-08 23:38:17,758 - 29835 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task:
Traceback (most recent call last):
  File "/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/hummingbot/client/command/history_command.py", line 79, in history_report
    cur_balances = await asyncio.wait_for(self.get_current_balances(market), network_timeout)
  File "/miniconda3/envs/hummingbot/lib/python3.8/asyncio/tasks.py", line 490, in wait_for
    raise exceptions.TimeoutError()
asyncio.exceptions.TimeoutError
```

I think nobody uses high number of orders and runs the bot for a long time.

**Comment by nikspz** [comment]
Should be fixed with #5600

**Comment by rapcmia** [comment]
Fix has been deployed
